.github/workflows/test-models.yml

```diff
@@ -30,8 +30,6 @@ jobs:
             suite: cpu
           - os: MacStudio
             suite: gpu
-          - os: MacStudio
-            suite: vulkan
```

Comment:
i would switch this to cpu so we don't run on Mac CPU anylonger. But it can be a follow on commit